### ISSUE TITLE
Question about baselines recurrent in paper

### ISSUE THREAD

**Issue by liuyufei-nubot** [Toytiny/RaFlow#4]
Thank you very much for such an excellent work of open source and I recommend it to many of my peers. I see the results of **SLIM** presented in your paper, but I have a problem with TF automatic gradient descent when reproducing this work, do you rewrite a set with **torch** or use their code directly, do you have the same problem?

**Comment by Toytiny** [comment]
Hi,

Thank you for your interest in our work! We reproduced SLIM ourselves using pytorch and didn't meet any gradient descent problem. If you're interested, we can discuss later. My email is : s2207427@ed.ac.uk

**Comment by liuyufei-nubot** [comment]
Thank you very much for your prompt reply, rewriting SLIM with pytorch is very beneficial work for the community, I have sent you an email and hope for more communication
